src/pybind/mgr/tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist = py27-{cov,run},py3-{cov,run}
+skipsdist = true
+toxworkdir = {env:CEPH_BUILD_DIR}/mgr
+minversion = 2.8.1
+
+[testenv]
+setenv=
+    CFLAGS = -DXMLSEC_NO_SIZE_T
+    UNITTEST = true
+    WEBTEST_INTERACTIVE = false
+    LD_LIBRARY_PATH = {toxinidir}/../../build/lib
+    PATH = {toxinidir}/../../build/bin:$PATH
+    py27: PYTHONPATH = {toxinidir}/../../build/lib/cython_modules/lib.2
+    py3:  PYTHONPATH = {toxinidir}/../../build/lib/cython_modules/lib.3
+    cov:  UNITTEST = true
+    cov:  COVERAGE_FILE = .coverage.{envname}
+commands=
+    pip install -r {toxinidir}/requirements.txt
```

Comment:
Would it be possible to use [`deps=`](https://tox.readthedocs.io/en/latest/config.html#conf-deps) here? always running `pip install` seems a bit heavy